includes/payment-tokens/class-wc-stripe-payment-tokens.php

```
@@ -356,6 +356,16 @@ function ( $payment_method ) {
 				) {
 					$token                      = $this->add_token_to_user( $payment_method, $customer, $payment_method_ids );
 					$tokens[ $token->get_id() ] = $token;
+
+					// If add_token_to_user updated an existing token's payment method ID (e.g. re-issued card),
+					// remove the stale old-ID key from $stored_tokens so the orphan-cleanup loop below
+					// does not delete the token that was just saved under the new ID.
+					foreach ( $stored_tokens as $old_pm_id => $stored_token ) {
```

PHPMD (2.15.0)

[error] The variable $old_pm_id is not named in camelCase. (undefined)

(CamelCaseVariableName)